pkg/ctl/utils/describe_addon_versions.go

```diff
@@ -29,7 +29,7 @@ func describeAddonVersionsCmd(cmd *cmdutils.Cmd) {
 	cmd.FlagSetGroup.InFlagSet("Addon", func(fs *pflag.FlagSet) {
 		fs.StringVar(&addonName, "name", "", "Addon name")
 		fs.StringVar(&k8sVersion, "kubernetes-version", "", "Kubernetes version")
-		fs.StringVarP(&clusterName, "cluster", "c", "", "EKS cluster name")
+		cmdutils.AddClusterFlag(fs, cmd.ClusterConfig.Metadata)
```

Comment:
This will break the command as `clusterName` is used below instead of `cmd.ClusterConfig.Metadata.Name`.

Reply:
ok, fixed.